src/lib/viewers/BaseViewer.js

```diff
@@ -811,8 +811,10 @@ class BaseViewer extends EventEmitter {
             return false;
         }
 
-        const canViewAnnotations = !!(permissions.can_view_annotations_all || permissions.can_view_annotations_self);
-        return !permissions.can_annotate && !canViewAnnotations;
+        const { can_annotate, can_view_annotations_all, can_view_annotations_self } = permissions;
+
+        // eslint-disable-next-line
+        return can_annotate || can_view_annotations_all || can_view_annotations_self;
```

Comment:
You were forcing a boolean before with !! on line 814, is ok to return undefined now if no permissions are passed in the object?

Reply:
Forcing it again now just to be sure :)